src/spec-node/containerFeatures.ts

```diff
@@ -105,6 +106,13 @@ export async function getExtendImageBuildInfo(params: DockerResolverParameters,
 	// Processes the user's configuration.
 	const featuresConfig = await generateFeaturesConfig(params.common, dstFolder, config.config, getContainerFeaturesFolder, additionalFeatures);
 	if (!featuresConfig) {
+		if (canAddLabelsToContainer && !imageBuildInfo.dockerfile) {
```

Comment:
I don't really understand `canAddLabelsToContainer`?  Is this strictly for compose?